runtime/near-vm-logic/src/logic.rs

```diff
@@ -2516,6 +2518,13 @@ impl<'a> VMLogic<'a> {
         let new_used_gas = self.gas_counter.used_gas();
         self.gas_counter.process_gas_limit(new_burn_gas, new_used_gas)
     }
+
+    #[cfg(feature = "sandbox")]
+    pub fn debug_log(&mut self, len: u64, ptr: u64) -> Result<()> {
```

Comment:
This is a public API, so it must be documented. Additionally, it's an API which is subtly different from all other APIs here (as it sandbox-only), so it needs specific docs for that as well.

Reply:
Let's move this function to after `storage_has_key`, to keep it with other host functions.